server/mem_agent/utils.py

```
@@ -0,0 +1,203 @@
+        needs_wrapping = True
+        for line in lines:
+            stripped = line.strip()
+            if (stripped.startswith(('def ', 'class ', 'import ', 'from ')) or 
+                stripped.startswith(('if ', 'for ', 'while ', 'try:', 'with ')) or
+                '=' in stripped or stripped.startswith(('print(', 'return '))):
+                needs_wrapping = False
+                break
+        
+        if needs_wrapping:
+            # Wrap in a function to make it valid Python for Black
+            wrapped_code = f"def temp_function():\n" + "\n".join(f"    {line}" for line in lines)
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)